machines/VCItemMachine/EsignetMosipVCItem/EsignetMosipVCItemMachine.ts

```diff
@@ -32,10 +41,8 @@ const model = createModel(
     verifiableCredential: null as VerifiableCredential,
     isPinned: false,
     hashedId: '',
-
     publicKey: '',
     privateKey: '',
-
```

Comment:
Looks like, by mistake deleted these

Reply:
@swatigoel empty lines?

Reply:
My bad, somehow I read it you removed publicKey and privateKey variables